docs/contract-test-collection.md

```diff
@@ -127,6 +129,21 @@ storage (pair
           (nat %total_supply));
 ```
 
+## LambdaViewWithStoredCounterContract
```

Comment:
I don't think this contract satisfy the tzip-4 view definition. Lambdaview in the name might be inaccurate:

> We define the following synonym in a contract's parameter type declaration:
> 
> ```
> view a r = (a, contract r)
> ```
> 
> A `view` is an entrypoint which represents a computation that takes an argument of type `a` and returns a result of type `r`. This return type is represented as a callback.

Reply:
I found the contract is the same as one originated in Live Code examples that is called "GenericMultisig"  so I have changed the name to "GenericMultisigContract"

Reply:
I have changed the name to "GenericMultiSig" as the contract is the same as the one in Taquito Live Code Examples.